deploy/v1/mpi-operator.yaml

```diff
@@ -0,0 +1,196 @@
+# --------------------------------------------------
+# - Single configuration deployment YAML for MPI-Operator
+# - Include:
+#      CRD deployment
+#      Namespace deployment
+#      RBAC deployment
+#      MPI-operator container deployment set
+# --------------------------------------------------
```

Comment:
Why is this file in the diff? Perhaps do a rebase to fix this?

Reply:
separate a new directory for different api version.